src/main/java/easytests/api/v1/mappers/SubjectsMapper.java

```diff
@@ -9,5 +12,15 @@
  */
 @Service("SubjectsMapperV1")
 public class SubjectsMapper extends ModelMapper {
+    public SubjectsMapper() {
+        super();
+        this.createTypeMap(SubjectModel.class, Subject.class)
+            .addMappings(
+                mapper -> mapper.when(
```

Comment:
do not copy paste code, that you don't understand. When here is not necessary, remove.